CHANGELOG.md

```diff
@@ -90,6 +90,11 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#4202](https://github.com/realm/SwiftLint/issues/4202)
 
+* Add `test_parent_classes` option to `balanced_xctest_lifecycle`, `single_test_class`
+  and `empty_xctest_method` rules.  
+  [Martin Redington](https://github.com/mildm8nnered)
+  [#4200](https://github.com/realm/SwiftLint/issues/4200)
+
```

Comment:
Please also mention in the "Breaking" section that `QuickSpec` is now considered a test case in all the rules by default.